packages/shell-api/src/replica-set.spec.ts

```diff
@@ -585,4 +587,151 @@ describe('ReplicaSet', () => {
       });
     });
   });
+
+  xdescribe('integration', () => {
```

Comment:
Doesn't `eslint` complain about this?

Should we rather use environment variables to control this? `process.env.MONGOSH_TEST_RS_HOST1` ...

Reply:
eslint does complain, but its a warning not a error. We can do the env variable and everything but it seems overkill if we're just going to fix the problem with CI soon enough.